examples/internal/proto/examplepb/a_bit_of_everything.swagger.json

```diff
@@ -6485,12 +6485,24 @@
     },
     "examplepbNumericEnum": {
       "type": "string",
+      "example": "\"ZERO\"",
       "enum": [
         "ZERO",
         "ONE"
       ],
       "default": "ZERO",
-      "description": "NumericEnum is one or zero.\n\n - ZERO: ZERO means 0\n - ONE: ONE means 1"
+      "description": "NumericEnum is one or zero.",
+      "title": "NumericEnum",
+      "externalDocs": {
+        "description": "Find out more about ABitOfEverything",
+        "url": "https://github.com/grpc-ecosystem/grpc-gateway"
+      },
+      "required": [
+        "uuid",
+        "int64Value",
+        "doubleValue"
+      ],
```

Comment:
This doesn't look right, this should only apply to the messages that contain these fields. A bug?

Reply:
It may be worthwhile for you to rebase on `main`, we just merged a PR that did some work around required fields.